dkakhanouski/01_HTML/Menu_Add.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html> 
+<HTML> 
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Меню</TITLE>
+    <style type="text/css">
+ul {
+  list-style: none;
+  margin: 0px;
+  padding: 0px;
+  border-bottom: 1px solid black;
+  border-radius: 5px;
+  width: 150px;
+}
+li {
+  position: relative;
+}
+li ul {
+  position: absolute;
+  opacity: 0;
+  width: 0;
+  left: 149px;
+  transition: all .5s ease-out;
+}
+li ul li:nth-of-type(1) {
+  margin-left: 0px;
+}
+li ul li:nth-of-type(2) {
+  margin-left: 0px;
+}
+li ul li:nth-of-type(3) {
+  margin-left: 0px;
+}
+li:hover ul {
```

Comment:
Is isn't good solution for smooth appearance of nested lists, you have hover zones under day of the week
nested list should be appear when we do hover just on the particular day

Reply:
I have fixed it, thanks.